docs/rollkit-stack.md

```diff
@@ -55,9 +51,9 @@ nonce/sequence number). This behaviour is similar to the Tendermint mempool.
 ### Light node
 
 Light nodes are the main producer of transactions in the Rollkit network.
-They participate in gossiping of and fraud proofs. Light nodes may only
-request or store a subset of the state, just to ensure that they can execute
-rollback.
+They participate in gossiping of fraud proofs. Light nodes may only
+request or store a subset of the state, so they can query chain balances
```

Comment:
We shouldn't put this here. Right now we have no state at all, and in the future, we will work on fancy trust-minimized approach for queries, but probably without explicit state storage.

Reply:
should we remove light node for now?

Reply:
No light nodes should be included. The statement is false.

Reply:
Could also be included somehow in such a way that describes it as a future feature / work in progress.

Reply:
@S1nus will expand on light nodes